diff --git a/plugins/styles/collapsible.inc b/plugins/styles/collapsible.inc
index 47949c9..dd437ab 100644
--- a/plugins/styles/collapsible.inc
+++ b/plugins/styles/collapsible.inc
@@ -127,7 +127,7 @@ function theme_panels_style_collapsible($vars) {
     
   $class = $vars['collapsed'] ? ' ctools-collapsed' : '';
   $output = '<div' . $pane_id . ' class="ctools-collapsible-container' . $class . $pane_css . '">';
-  $output .= '<h2 class="pane-title ctools-collapsible-handle">' . check_plain($vars['handle']) . '</h2>';
+  $output .= '<h2 class="pane-title ctools-collapsible-handle">' . $vars['handle'] . '</h2>';
   $output .= '<div class="ctools-collapsible-content">' . $vars['content'] . '</div>';
   $output .= '</div>';
 
